### PR TITLE
Render results of sorting list

### PR DESCRIPTION
**Pull request by vokarl**
- just rendered the preliminary result list at the bottom of the result page

### CODE REVIEW

File: components/Form/Index.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ import React from "react";
 let inputNumber = 1;
 
 const RenderInputField = ({ label }) => {
-  const inputId = `input$(inputNumber)`;
+  const inputId = `input${inputNumber}`;
   inputNumber++;
+  console.log("Input ID:", inputId);
```
Comment (julialoeschel) [inline components/Form/Index.js:8]: For every PR we delete the console logs :) they dont belong on main 
(except error logs)

File: pages/sortinglist/[slug].js [hunk 2]
```diff
@@ -166,6 +175,17 @@ export default function DynamicForm() {
...
       </StyledForm>
+
+      <StyledList>
+        <h2>Ergebnisliste:</h2>
+        {Object.entries(recentTreeData)
+          .filter(([key]) => !key.includes("attributes"))
+          .map(([key, value]) => (
+            <ListItem key={key}>
+              {key}: {value}
+            </ListItem>
```
Comment (julialoeschel) [inline pages/sortinglist/[slug].js:186]: good job!

**Review by julialoeschel — COMMENTED** [review]
good Job!

very well done, 
🐜

File: components/Form/Index.js (hunk 1 above)
Comment (vokarl) [reply components/Form/Index.js:8]: Oops😅 I missed that one. Thanks!

**Comment by julialoeschel** [comment]
👁️  LGTM

**Pull request merged**